karapace/schema_models.py

```diff
@@ -264,9 +325,10 @@ def __init__(
         schema: Draft7Validator | AvroSchema | ProtobufSchema,
         references: Sequence[Reference] | None = None,
         dependencies: Mapping[str, Dependency] | None = None,
+        schema_wrapped: Draft7Validator | AvroSchema | ProtobufSchema | None = None,
```

Comment:
This and it's respective instance variables are not used anywhere in the code, why are you adding this?

Reply:
Seems it must be removed.